WcaOnRails/app/models/round.rb

```diff
@@ -33,22 +38,21 @@ def final_round?
   end
 
   def multibld_attempt_to_points(attempt_value)
-    solve_time = SolveTime.new("333mbf", :best, attempt_value)
-    solve_time.points
+    SolveTime.new("333mbf", :best, attempt_value).points
```

Comment:
Making this a static method of `SolveTime` would allow us to reuse the method =)